tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/matchstmt/varbindingpatternmatchpattern/ErrorBindingPatternTest.java

```diff
@@ -42,6 +50,16 @@ public void setup() {
                 "/error_binding_pattern_negative.bal");
     }
 
+    @Test
+    public void testNegativeSemantics() {
+        CompileResult buildError = BCompileUtil.compile("test-src/statements/matchstmt/" +
+                "varbindingpatternmatchpattern/error_type_ref_negative.bal");
```

Comment:
```suggestion
        CompileResult buildError = BCompileUtil.compile(
                "test-src/statements/matchstmt/varbindingpatternmatchpattern/error_type_ref_negative.bal");
```

Just to help with navigating to the file.